tests/baselines/reference/transpile/declarationRestParameters.d.ts

```diff
@@ -31,17 +48,30 @@ export declare const v2: (n: "n", a: "a") => {
 
 //// [Diagnostics reported]
 v2.ts(2,14): error TS2527: The inferred type of 'v2' references an inaccessible 'unique symbol' type. A type annotation is necessary.
+v2.ts(4,15): error TS9039: Type containing private name 'a' can't be used with --isolatedDeclarations.
+v2.ts(6,5): error TS9013: Expression type can't be inferred with --isolatedDeclarations.
+v2.ts(6,15): error TS9039: Type containing private name 'n' can't be used with --isolatedDeclarations.
 
 
-==== v2.ts (1 errors) ====
+==== v2.ts (4 errors) ====
     const n = Symbol();
     export const v2 = (...a: [n: "n", a: "a"]): {
                  ~~
 !!! error TS2527: The inferred type of 'v2' references an inaccessible 'unique symbol' type. A type annotation is necessary.
         /** r rest param */
         a: typeof a,
+                  ~
+!!! error TS9039: Type containing private name 'a' can't be used with --isolatedDeclarations.
+!!! related TS9027 v2.ts:2:14: Add a type annotation to the variable v2.
         /** module var */
         n: typeof n,
+        ~
+!!! error TS9013: Expression type can't be inferred with --isolatedDeclarations.
+!!! related TS9027 v2.ts:2:14: Add a type annotation to the variable v2.
+!!! related TS9035 v2.ts:6:5: Add satisfies and a type assertion to this expression (satisfies T as T) to make the type explicit.
+                  ~
+!!! error TS9039: Type containing private name 'n' can't be used with --isolatedDeclarations.
+!!! related TS9027 v2.ts:2:14: Add a type annotation to the variable v2.

```

Comment:
This is all new an interesting but I think still valid.

Reply:
I think these errors are redundant. I'll suppress them

Reply:
I don't think it's a requirement; the errors seem correct to me?

Reply:
Yes. It's correct. It's pointing out the exact parts that are causing the issue.

Reply:
Yeah, I wouldn't worry about suppressing them; my original comment wasn't meant to be a complaint! 😅